e2e_tests/tests/cluster/test_workspace_org.py

```diff
@@ -18,6 +18,9 @@
 @pytest.mark.e2e_cpu
 def test_workspace_org() -> None:
     master_url = conf.make_master_url()
+    # Ensure determined creds are the default values.
+    with logged_in_user(ADMIN_CREDENTIALS):
+        change_user_password("determined", "")
```

Comment:
maybe having a fixture to do this is better? Instead of having sprinkles everywhere.

Reply:
if we change default determined password in another test, which causes it to be broken here, we could reset it back in that test (e.g. in `finally` clause)?